pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -971,6 +977,13 @@ private List<PdfConfigInfo> findPdfConfigsForStudyUser(List<PdfConfigInfo> study
             }
         }
 
+        //lms special case to handle scenarios where lms user consented-v1 but had release-v2
+        if (!studyConfigs.isEmpty() && studyConfigs.get(0).getStudyGuid().equals(LMS_GUID)) {
```

Comment:
special handling of pdf config with users consented - v1 and completed newer release - v2